docs/sources/Cisco/index.md

```diff
@@ -438,3 +438,60 @@ index=netops sourcetype=cisco:wsa:*
 ```
 
 Verify timestamp, and host values match as expected
+
+## Product - ESA
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| Splunk Add-on  | https://splunkbase.splunk.com/app/1761/                                                                |
+| Product Manual | https://www.cisco.com/c/en/us/td/docs/security/esa/esa13-5-1/user_guide/b_ESA_Admin_Guide_13-5-1.html |
+
+* Update ``vi /opt/sc4s/local/context/vendor_product_by_source.conf `` update the host or ip mask for ``f_cisco_esa`` to identiy the esa events.
```

Comment:
this goes under ``### Setup and Configuration

Reply:
Moved inside the ### Setup and Configuration